blogposts/2021/dev-tool-time-primeagen.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Thinking less to do more: Dev Tool Time with ThePrimeagen'
+description: 'ThePrimeagen, Senior Software Engineer at Netflix and Twitch Partner tech streamer, shares how he reduces cognitive overhead with tmux, i3, and Neovim to write code more effectively.'
+author: Scott Bailey
+authorUrl: https://about.sourcegraph.com/handbook/company/team#scott-bailey-he-him
+publishDate: 2021-11-03T10:00-07:00
+tags: [blog]
+slug: dev-tool-time-theprimeagen
+heroImage: https://sourcegraphstatic.com/blog/dev-tool-time/primeagen/linkedin.jpg
+socialImage: https://sourcegraphstatic.com/blog/dev-tool-time/primeagen/linkedin.jpg
+published: true
+---
+
+As a popular Twitch streamer and Vim enthusiast, ThePrimeagen is known for his fast-paced tmux and Neovim focused workflow, shifting quickly between windows, buffers, and code chunks. When he joined us for [Dev Tool Time](https://info.sourcegraph.com/dev-tool-time) on October 27, 2021, he shared his approach to reducing cognitive overhead in his dev workflow, and showed how his custom configurations make it easier to think less and do more.
+
+You can watch the recording of this episode at the [bottom of this blog post](#Watch-the-recording), but first let’s check out ThePrimeagen’s desk setup:
+
+## Hardware
+
+![ThePrimeagen's current desk setup](https://sourcegraphstatic.com/blog/dev-tool-time/primeagen/hardware.jpg)
+
+ThePrimeagen's desk setup is doubly determined by his work as a developer and his position as a streamer. While he prefers coding on a single, relatively normal-sized monitor, he has two on his desk. The [Acer Predator](https://www.acer.com/ac/en/US/content/predator-models/monitors) is his primary monitor, attached to his [System76](https://system76.com/) laptop running Ubuntu. The second monitor is hooked up to a Windows machine used for running his Twitch stream, since some of his hardware, such as the [Streamdeck](https://www.elgato.com/en/stream-deck) and [GoXLR](https://www.tc-helicon.com/product.html?modelCode=P0CQK), don't yet have Linux drivers. [Kinesis](https://kinesis-ergo.com/) keyboards, notably the [Advantage 2](https://kinesis-ergo.com/shop/advantage2/) take up a chunk of desk space. As part of his drive to make typing more efficient, ThePrimeagen uses the Dvorak keyboard layout, which optimizes for alternating keypresses between the two hands and a focus on the home row among other efficiencies.
```

Comment:
```suggestion
ThePrimeagen's desk setup is doubly determined by his work as a developer and his position as a streamer. While he prefers coding on a single, relatively normal-sized monitor, he has two on his desk. The [Acer Predator](https://www.acer.com/ac/en/US/content/predator-models/monitors) is his primary monitor, which is attached to his [System76](https://system76.com/) laptop running Ubuntu. The second monitor is hooked up to a Windows machine that runs his Twitch stream, since some of his hardware, such as the [Streamdeck](https://www.elgato.com/en/stream-deck) and [GoXLR](https://www.tc-helicon.com/product.html?modelCode=P0CQK), don't yet have Linux drivers. [Kinesis](https://kinesis-ergo.com/) keyboards, notably the [Advantage 2](https://kinesis-ergo.com/shop/advantage2/) take up a chunk of desk space. 

As part of his drive to make typing more efficient, ThePrimeagen uses the Dvorak keyboard layout, which optimizes for alternating keypresses between two hands and focuses on the home row among other efficiencies.
```